doc/classes/AnimationNodeBlendSpace2D.xml

```diff
@@ -95,6 +108,7 @@
 			<argument index="1" name="node" type="AnimationRootNode">
 			</argument>
 			<description>
+				Changes the AnimationNode referenced by the point at index [code]point[/code].
```

Comment:
Should it be `AnimationRootNode` as per the argument type?

Reply:
Yes. I wrote AnimationNode because AnimationRootNode doesn't have any public methods or properties, and extends AnimationNode. So if you go check its docs there's little to learn. But I'll change it now.